<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetPID</class>
 <widget class="QDialog" name="DialogSetPID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Apply">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>P</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>SetPoint</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditP"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditI"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditD"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditSP"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting PID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetPID</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
